scripts/us_usda/quickstats/README.md

```diff
@@ -1,3 +1,16 @@
+The `process.py` script generates statvars based on the 
+USDA agriculture survey (and census data very soon) data 
+using the USDA quickstats [API][api].
+
+[api]: https://quickstats.nass.usda.gov/api/
```

Comment:
Could you pls add a note that running this will require a key with a link to get the key and config file to set it.

Reply:
Done.